README.md

```diff
@@ -479,6 +482,45 @@ const transaction = arweave.transactions.get('bNbA3TEQVL60xlgCcqdz4ZPHFZ711cZ3hm
 });
 ```
 
+### Blocks
+Blocks are base elements of Arweave's blockweave data structure.
+Each block is linked to two prior blocks: the previous block in the "chain" (as with traditional blockchain
+protocols), and a block from the previous history of the blockchain (the "recall block"). Each block contains
+a list of zero to many transactions.
+
+
+#### Get a block by indep_hash
+Gets block data for given independent hash (see page 63. of https://www.arweave.org/yellow-paper.pdf for details)
```

Comment:
Can we make this a href rather than the raw URL please?

Reply:
fixed